test/e2e/windows/memory_limits.go

```diff
@@ -96,7 +96,7 @@ func checkNodeAllocatableTest(ctx context.Context, f *framework.Framework) {
 	ginkgo.By(fmt.Sprintf("Checking stated allocatable memory %v against calculated allocatable memory %v", &nodeMem.allocatable, calculatedNodeAlloc))
 
 	// sanity check against stated allocatable
-	framework.ExpectEqual(calculatedNodeAlloc.Cmp(nodeMem.allocatable), 0)
+	gomega.Expect(calculatedNodeAlloc.Cmp(nodeMem.allocatable)).To(gomega.Equal(0), "calculated allocatable memory and stated allocatable memory are same")
```

Comment:
Here it makes sense to include the actual values in the annotation. `To` supports formatting.

With that change, the `By` above becomes pretty useless. I suggest removing it.